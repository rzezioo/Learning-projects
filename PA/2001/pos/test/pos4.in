40 135
35
95
25
8
10
98
52
45
25
6
55
46
46
52
61
7
9
43
92
79
84
28
52
75
12
100
58
56
83
64
22
47
94
78
40
22
10
63
34
46
2 19
20 20
8 9
8 3
39 40
35 13
25 26
9 13
37 21
2 21
34 2
22 38
36 12
27 28
40 1
32 40
9 3
6 6
10 19
30 28
20 39
12 13
25 3
29 9
11 3
18 6
22 15
17 18
23 24
4 5
10 10
29 30
31 13
18 19
15 15
35 40
16 17
9 20
3 4
8 4
13 14
21 25
28 11
8 8
33 24
31 19
21 16
31 34
4 27
36 35
30 31
9 10
5 38
17 16
18 18
21 22
37 38
36 37
9 9
21 19
3 14
10 11
2 3
27 26
32 33
11 25
32 2
38 38
34 35
4 4
31 32
25 18
13 29
36 18
31 40
22 24
33 34
4 36
20 34
35 14
32 35
17 11
15 13
11 23
16 13
26 27
36 36
1 2
6 7
28 29
11 2
30 1
12 25
32 10
14 15
28 26
8 6
19 37
38 30
23 23
30 19
35 28
20 21
7 8
8 30
15 16
10 40
22 23
23 22
14 14
24 25
34 32
15 16
22 28
15 10
16 16
35 36
11 12
33 4
40 38
23 32
1 5
14 22
30 31
28 39
25 18
38 39
3 6
5 6
19 20
23 2
20 21
34 34
14 6
40 31
