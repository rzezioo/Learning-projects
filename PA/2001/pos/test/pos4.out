135
1 40 39 20 20 9 8 8 3 9 29 30 28 11 3 25 21 2 19 10 10 9 9 13 35 36 12 13 31 34 35 40 32 33 34 2 32 31 19 18 18 17 16 17 11 25 18 6 6 8 4 3 14 13 29 28 27 4 4 5 38 22 15 15 13 16 21 37 36 36 4 33 24 23 23 11 10 32 35 14 14 15 16 16 15 10 40 31 30 1 2 11 12 25 26 27 26 28 35 36 18 25 24 22 23 22 21 19 37 38 38 30 19 20 21 20 34 34 32 23 2 3 6 7 8 30 31 40 38 39 28 22 14 6 5 1